distributed/distributed-schema.yaml

```diff
@@ -404,6 +404,49 @@ properties:
                   non-time-sensitive heuristics. This should be set to be longer than
                   the duration of most dask tasks.
 
+              rebalance:
+                type: object
+                description: >-
+                  Settings for data rebalance operations
+                properties:
+                  measure:
+                    enum:
+                      - process
+                      - optimistic
+                      - managed
+                      - managed_in_memory
+                    description: >-
+                      Which of the properties of distributed.scheduler.MemoryState
+                      should be used for measuring worker memory usage
+                  sender_min:
```

Comment:
Small note: most configuration names today use hyphens by default.  I think that get/set are intentionally blind to this difference, but we may want to stay consistent

Reply:
ok will change